apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.html

```diff
@@ -110,7 +114,7 @@
                       <div *ngIf="slot.tags_list" class="tags">
                         <app-badge
                           *ngFor="let tag of splitTags(slot.tags_list)"
-                          [fontSize]="'small'"
+                          [fontSize]="'extraSmall'"
```

Comment:
This will make a conflict , on another PR you name it as mini in same component and here it is extrasmall

And name should be extra-small or xs

Reply:
x-small